### PR TITLE
Using Parsers.Common to find the projects

### PR DESCRIPTION
**Pull request by MichielOda**
Added option to specify a specific solution (in case unit tests contain solutions). Removed ProjectFile and subsequently the unit test project as we make use of the info from Parsers.

### CODE REVIEW

File: CICD.Tools.SDKChecker/Program.cs [hunk 1]
```diff
@@ -47,73 +46,53 @@ public static async Task<int> Main(string[] args)
                 IsRequired = false
             };
 
+            var solutionFilePath = new Option<string>(
+                name: "--solution-filepath",
+                description: "The filepath to the solution.");
```
Comment (janstaelensskyline) [inline CICD.Tools.SDKChecker/Program.cs:51]: Would indicate this as Optional, for developer clarity IsRequired = false.

File: CICD.Tools.SDKChecker/Program.cs [hunk 2]
```diff
@@ -47,73 +46,53 @@ public static async Task<int> Main(string[] args)
...
-            if (String.IsNullOrWhiteSpace(pathToSolution))
+            List<string> legacyStyleProjects = new List<string>();
+            foreach (var projectInSolution in solution.Projects)
             {
-                throw new InvalidOperationException("Could not located a solution file (.sln) in workspace: " + workspace);
-            }
+                var project = solution.LoadProject(projectInSolution);
 
-            var projectsWithPackageConfig = RetrieveLegacyStyleProjects(pathToSolution);
+                if (project is not { ProjectType: ProjectType.Sdk })
```
Comment (janstaelensskyline) [inline CICD.Tools.SDKChecker/Program.cs:89]: If parsers was changed, this may need to become ProjectStyle

Comment (MichielOda) [reply CICD.Tools.SDKChecker/Program.cs:89]: Done

File: CICD.Tools.SDKChecker/Program.cs (hunk 1 above)
Comment (MichielOda) [reply CICD.Tools.SDKChecker/Program.cs:51]: Done

**Pull request merged**